spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
+  let(:orphan_statuses) { OrphanStatus.all }
+  let(:orphan_sponsorship_statuses) { OrphanSponsorshipStatus.all }
+  let(:provinces) { Province.all }
+  let(:orphan) { build_stubbed :orphan }
+
+  before :each do
+    assign(:orphan_statuses, orphan_statuses)
+    assign(:orphan_sponsorship_statuses, orphan_sponsorship_statuses)
+    assign(:provinces, provinces)
+    assign(:orphan, orphan)
```

Comment:
Minor point: in view specs, when a variable is used locally in the spec and is assigned as an instance variable in the view under test, you can save some space by removing the `let`s and changing the `before` block to
```ruby
before :each do
  @orphan_statuses = OrphanStatus.all
  @provinces = Province.all
  .
  .
end
```